plugins/package-managers/go/src/funTest/kotlin/GoDepFunTest.kt

```diff
@@ -42,7 +42,7 @@ class GoDepFunTest : WordSpec({
     "GoDep" should {
         "resolve dependencies from a lockfile correctly" {
             val definitionFile = getAssetFile("projects/synthetic/godep/lockfile/Gopkg.toml")
-            val expectedResultFile = getAssetFile("projects/synthetic/godep-expected-output.yml")
+            val expectedResultFile = getAssetFile("projects/synthetic/lockfile-expected-output.yml")
```

Comment:
I'm not sure about this, as this looses the information about which package manager this file belongs to. Maybe just use "lockfile" as an infix?

Reply:
> this looses the information about which package manager this file belongs to.

Ah, you're solving that in the next commit by introducing subdirectories, ok.